cogs/gambling.py

```diff
@@ -324,7 +324,7 @@ async def wheel(self, ctx: mido_utils.Context, amount: Union[int, str]):
         e.description = f"**{ctx.author}** has just won: " \
                         f"**{mido_utils.readable_currency(won_cash)}** [x{won_multiplier}]\n\n"
         for i, multiplier_and_arrow in enumerate(possibilities_and_arrows.items()):
-            multiplier, arrow = multiplier_and_arrow
+            multiplier = multiplier_and_arrow   # removed , arrow since it was unused 
```

Comment:
have you tested this change? `multiplier_and_arrow` contains both the multiplier and the arrow emote. the `multiplier` variable then gets used in the description. with this change, the wheel description would be broken. 

but good catch, `arrow` isn't used here. in that case, we can just iterate over only the multipliers, which can be done by `for i, multiplier in enumerate(possibilities_and_arrows.keys()):`